blotztask-api/Models/MonthlyStatDTO.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+
+namespace BlotzTask.Models
+{
+    public class MonthlyStatDTO
+    {
+        public MonthlyStatDTO(string inputMonth) {
+            CurrentMonth = inputMonth;
+            Data = new Tasks();
+        }
+        public string CurrentMonth { get; set; }
+        public Tasks Data { get; set; }
```

Comment:
```suggestion
        public Tasks Tasks { get; set; }
```

Data does not give any specific meaning to the object , ⬇️ readability